integration/k8s_custom_deploy_test.go

```diff
@@ -62,7 +62,7 @@ func TestK8sCustomDeploy(t *testing.T) {
 	defer cancel()
 	f.CurlUntil(ctx, "http://localhost:54871", "Greetings from Live Update!")
 
-	f.Touch(filepath.Join("web", "fallback.txt"))
+	f.Touch("fallback.txt")
```

Comment:
maybe we can also add a check here that the pod ID didn't change / an image build didn't start?

Reply:
Hmm yeah let me see what we've got handy here 🔨

Reply:
Ah you know what, this test is actually already set up in a way that it'll fail if the pod is (or isn't) replaced based on what it's going.

It's kind of a "file sync only" setup in that it's using `nginx` and then live updates in a file, and the `apply_cmd` is `kubectl delete && kubectl create` so that we always destroy things.